index.html

```diff
@@ -0,0 +1,118 @@
+        </div>
+        <p class="venue"><span class="dates">2022.10.26(WED) ~ 28(FRI)</span> <br> @Accra International Conference Centre</p>
+    </section>
+    <section id="highlights" class="container">
+        <h3 class="highlights-title">Main Program</h3>
+        <hr class="divider">
+        <div class="activity-items">
+            <div class="box-h-item flex-activities">
+                <img src="img/conference.png" alt="">
+                <h3>Panel Discustions</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/workshop.png" alt="">
+                <h3>Workshops</h3>
+                <div>
+                    <p>Extrapreneuring Our Way into Digital Transformation</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/speech-bubble.png" alt="">
+                <h3>Panel Discustions</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/exhibition.png" alt="">
+                <h3>Training Session</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/fireball.png" alt="">
+                <h3>Expert Session</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+        </div>
+        <div class="cta-highlights">
+            <a href="#">Join GDIW 2022</a>
+        </div>
+        <a href="#" class="cta-highlights-desktop">See the whole Program</a>
+    </section>
+    <section id="speakers" class="container">
+    </section>
+    <section id="partner" class="container">
+        <h3>Partners</h3>
+        <hr class="divider">
+        <div class="logos">
+            <img src="img/Google-Logo.wine.png" alt="GIZ Ghana">
+            <img src="img/Airbnb-Logo.wine.png" alt="Ministry of Communication">
+            <img src="img/Stripe_(company)-Logo.wine.png" alt="Embassy of Israel">
+            <img src="img/Facebook-Logo.wine.png" alt="Ghana Hubs Network">
+        </div>
+    </section>
+    <footer class="container">
+        <img src="img/Nov-GDIW-LOGO.svg" alt="GDIW 2022 Logo">
+        <div>
+            <p>THE GDIW 2022 | All Rights Reserved</p>
+        </div>
+    </footer>
```

Comment:
Kindly make sure that your project sticks as much as possible to the template provided. The footer section should contain a logo aligned to the left and text aligned to the right. Please fix this on both pages. :eyes: :eyes: 

Your project
![image](https://user-images.githubusercontent.com/98363075/196596276-31c125c2-116c-4ef1-8026-45a3ea42451a.png)


Template provided
![image](https://user-images.githubusercontent.com/98363075/196596204-d4154d71-aaaa-4ed5-a658-f62b18ab4190.png)